docs/StardustDocs/topics/createDataFrame.md

```diff
@@ -213,3 +215,28 @@ val df = students.toDataFrame {
 ```
 
 <!---END-->
+
+### DynamicDataFrameBuilder
+
+Previously mentioned dataframe constructors throw an exception when column names are duplicated. 
+When implementing a custom operation involving multiple dataframes, computed columns or parsing some third-party data,
```

Comment:
difficult sentence mixing two summations. I'd rephrase it like:

"When implementing a custom operation involving multiple dataframes, or computed columns or when parsing some third-party data..."

Reply:
Indeed, thank you c: